### PR TITLE
Failing test for #2390

### PR DESCRIPTION
**Pull request by silentroach**
Here is a failing test for #2390 (`FindOperator` is trying to be transformed instead of value in `find`)

### CODE REVIEW

**Comment by pleerock** [comment]
Can you please make this PR against `next` branch? I most probably will fix this issue on that branch.

File: test/github-issues/2390/entity/Post.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import {Column, Entity, PrimaryColumn} from "../../../../src";
+import {ValueTransformer} from "../../../../src/decorator/options/ValueTransformer";
+
+class Transformer implements ValueTransformer {
+
+    to(value: string): string {
+        // here we expect string only, not "In" FindOperator
+        return value.substring(1);
```
Comment (silentroach) [inline test/github-issues/2390/entity/Post.ts]:
```js
TypeError: value.substring is not a function
```

**Comment by silentroach** [comment]
done

**Comment by pleerock** [comment]
Thank you. Let's just keep this PR opened until I return to this task. Strange that it didn't run any tests...

**Pull request closed without merging**